src/styles/index.scss

```diff
@@ -0,0 +1,12 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@import "./card.scss";
+@import "./stars.scss";
+@import "../utils/variables";
+
+* {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
- Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your
web document.

- Apply font-family to `html` tag and inherit where needed